From 38e8fab678397c9e5a8298ce332461d2a6a4bc71 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Thu, 30 May 2024 17:38:30 +0200
Subject: [PATCH 0862/1695] dts: stm32: add I2C3 pinmux for stm32mp215f-dk
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add pinmux for I2C3 in pinctrl device-tree file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git fef4652da1f76600210f4065e9a1ac40cccf2c6f dts: stm32: add I2C3 pinmux for stm32mp215f-dk]
Change-Id: Ic5ff0ded9024f7d476c03371933c12f258416d93
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/384147
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp21-pinctrl.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp21-pinctrl.dtsi b/core/arch/arm/dts/stm32mp21-pinctrl.dtsi
index c4c6d00bd..8090af4a3 100644
--- a/core/arch/arm/dts/stm32mp21-pinctrl.dtsi
+++ b/core/arch/arm/dts/stm32mp21-pinctrl.dtsi
@@ -7,6 +7,16 @@
 #include <dt-bindings/pinctrl/stm32-pinfunc.h>
 
 &pinctrl {
+	i2c3_pins_a: i2c3-0 {
+		pins1 {
+			pinmux = <STM32_PINMUX('Z', 1, AF8)>, /* I2C3_SCL */
+				 <STM32_PINMUX('Z', 0, AF8)>; /* I2C3_SDA */
+			bias-disable;
+			drive-open-drain;
+			slew-rate = <0>;
+		};
+	};
+
 	usart2_pins_a: usart2-0 {
 		pins1 {
 			pinmux = <STM32_PINMUX('A', 4, AF6)>; /* USART2_TX */
-- 
2.47.2

